lib/matplotlib/figure.py

```diff
@@ -1882,6 +1921,14 @@ def subplot_mosaic(self, mosaic, *, sharex=False, sharey=False,
         # special-case string input
         if isinstance(mosaic, str):
             mosaic = self._normalize_grid_string(mosaic)
+            per_subplot_kw = {
+                tuple(k): v for k, v in per_subplot_kw.items()
+            }
+
+        per_subplot_kw = self._norm_per_subplot_kw(
+            per_subplot_kw
+        )
```

Comment:
This fits in a line?
```suggestion
        per_subplot_kw = self._norm_per_subplot_kw(per_subplot_kw)
```

Reply:
The place-holder name probably made this too long.